internal/test/helpers.go

```diff
@@ -105,66 +105,41 @@ func TestRoute(t *testing.T, api API, external bool, method, path, body string,
 		t.Skip("This test is for internal testing only")
 	}
 
-	if resp.StatusCode != expectedStatus {
-		t.Errorf("%s: expected status %v, but got %v", path, expectedStatus, resp.StatusCode)
-	}
+	require.Equalf(t, expectedStatus, resp.StatusCode, "SendHTTP failed for path %s", path)
 
 	replyJSON, err := ioutil.ReadAll(resp.Body)
-	if err != nil {
-		t.Errorf("%s: could not read response body: %v", path, err)
-		return
-	}
+	require.NoErrorf(t, err, "%s: could not read response body", path)
 
 	if expectedJSON == "" {
-		if len(replyJSON) != 0 {
-			t.Errorf("%s: expected no response body, but got:\n%s", path, replyJSON)
-		}
-		return
+		require.Equalf(t, 0, len(replyJSON), "%s: expected no response body, but got:\n%s", path, replyJSON)
```

Comment:
`require.Lenf` can be used here, but I don't mind using `Equalf`, just wanted to mention that.

Reply:
Thanks for pointing this out. I like it better.